dependencies/bionic/hammer_cli_foreman_puppet/changelog

```diff
@@ -0,0 +1,5 @@
+ruby-hammer-cli-foreman-puppet (0.0.2-1) UNRELEASED; urgency=medium
```

Comment:
I'm wondering about the UNRELEASED part. Is that needed?

Reply:
I have no clue, just used the script :(

Reply:
It seems like it should be `stable` as in other plugins...